app/views/projects/_index_dashboard.html.erb

```diff
@@ -2,144 +2,200 @@
+  margin: 30px;
+  margin-top: 0;
   float: left;
+  border-radius: 2px;
 }
 
-.todo .project .number a {
-  color: inherit;
-  text-decoration: inherit;
+.todo .circle {
+  margin-bottom: 0px;
 }
 
-.todo .project .release {
-  float: left;
+.released-project h2 {
+  padding: 0;
+  margin: 0;
+  margin-top: -4px;
 }
 
-.todo .project .who {
-  float: left;
-  clear: left;
-  margin-left: 5px
+.released-project p {
+  margin-top: 3px;
+  color: #666666;
 }
 
 </style>
 
-<svg width="293px" height="59px" viewBox="0 0 293 59" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
-        <g id="Artboard" transform="translate(-26.000000, -62.000000)">
-            <text id="Commits-since-last-d" font-family="Unica77LLWebRegular" font-size="20" font-weight="400" fill="#000000">
-                <tspan x="53.1" y="99">Commits since last deploy</tspan>
-            </text>
-            <rect id="Rectangle" stroke="#000000" stroke-width="2" x="27" y="63" width="291" height="57" rx="6"></rect>
-        </g>
-    </g>
-</svg>
+
+<h1>Out of sync</h1>
 
 <div class="todo">
-  <% @projects.reject(&:fully_released?).each do |project| %>
-    <div class="project">
-      <h2><div class="red-circle"></div><%= project.name.titleize %></h2>
+  <% @todo = @projects.reject(&:fully_released?).sort_by { |p| p.snapshot&.comparisons&.map { |c| c.description.length }.sum }.reverse %>
+  <% @todo.each do |project| %>
+    <% commits = project.snapshot&.comparisons&.map { |c| c.description.length }.sum  %>
+    <div class="project <%= commits > 10 ? "red" : ""%> ">
+      
+      <h2><%= project.name.titleize %></h2>
+      <p><%= project.description %></p>
+      <div class="comparisons">
         <% project.stages.ordered.each_cons(2) do |ahead, behind| %>
-          <% comparison = project.snapshot&.comparisons&.detect{|c| c.behind_stage_id == behind.id && c.ahead_stage_id == ahead.id } %>
-          <% if comparison && !comparison.released? %>
+          <% comparison = project.snapshot&.comparisons&.detect{ |c| c.behind_stage_id == behind.id && c.ahead_stage_id == ahead.id } %>
+          <% if comparison %>
             <div class="comparison">
-              <div class="number"><%= link_to comparison.description.length, params.permit(:organization_id).merge(view: 'detail', anchor: project_anchor(project)), 'data-turbolinks' => false %></div>
-              <div class="release">
-                <%= ahead.name.titleize %> to<br /><%= behind.name.titleize %><br />
-              </div>
-              <div class="who">
-                <%= comparison.description.map { |line| first_name_from_log(line).titleize }.uniq.to_sentence %>
-              </div>
+              <% commit_length = comparison.description.length %>
+              <% circle_color = commit_length == 0 ? "green" : commit_length > 10 ? "red": "yellow" %>
```

Comment:
Now that I'm looking more closely, I'm not sure I agree with this classification... it's more about the stage (e.g., failing a master build) or age (e.g., undeployed for X days) than the number of commits.

Either way we can refine this later. (When we do, this could be a helper rather than repeat the logic throughout the template.)